examples/BUILD

```diff
@@ -18,6 +18,7 @@ filegroup(
         ],
         exclude = [
             "cache/responses.yaml",
+            "dynamic-config-fs/**/*",
```

Comment:
this isnt ideal, but theres not an obvious way to include the dynamic `yaml` without it being treated as a standalone envoy config

Reply:
Hmm yeah. I think we could probably config check these separately somehow but not a huge deal for now.